Sources/xcodeproj/Objects/Project/PBXProj.swift

```diff
@@ -212,16 +212,16 @@ extension PBXProj: Equatable {
 // MARK: - Writable
 
 extension PBXProj: Writable {
-    public func write(path: Path, override: Bool) throws {
-        try write(path: path, override: override, outputSettings: PBXOutputSettings())
+    public func write(path: Path, override: Bool) {
+        write(path: path, override: override, outputSettings: PBXOutputSettings())
     }
 
-    public func write(path: Path, override: Bool, outputSettings: PBXOutputSettings) throws {
+    public func write(path: Path, override: Bool, outputSettings: PBXOutputSettings) {
```

Comment:
Probably, as said, this is *too* far, writes can definitely fail, and it is a “runtime error” rather than “programmer error” if that should happen.